include/mgmt/hawkbit/config_data.h

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2021 Advanced Climate Systems
+ *
+ * SPDX-License-Identiier: Apache-2.0
+ */
+#ifndef ZEPHYR_INCLUDE_MGMT_HAWKBIT_CONFIG_DATA_H_
+#define ZEPHYR_INCLUDE_MGMT_HAWKBIT_CONFIG_DATA_H_
+#include <autoconf.h>
```

Comment:
I think `autoconf.h` is automatically included?

Reply:
Should be addressed now